Epam.TAF.Web/PageObgects/Pages/EpamContinuumPage.cs

```diff
@@ -0,0 +1,22 @@
+﻿using Epam.TAF.Core.BasePage;
+using Epam.TAF.Core.Elements;
+using OpenQA.Selenium;
+
+namespace Epam.TAF.Web.PageObgects.Pages
+{
+    public class EpamContinuumPage : BasePage
+    {
+        private const string _breadCrumbsEpanContinuumPageXPath = "//*[@href='/services/consult-and-design' and @class='breadcrumbs__link']";
+
+        public override string Url => "https://www.epam.com/services/consult-and-design";
```

Comment:
better to move urls to some addition test data file or configuration file.

Reply:
Created file with URL. But I think my implementation it's not better solution.